lib/galaxy/files/sources/webdav.py

```diff
@@ -3,12 +3,12 @@
 except ImportError:
     WebDAVFS = None
 
-import tempfile
 from typing import (
     Optional,
     Union,
 )
 
+from galaxy import config
```

Comment:
I don't think this will work :thinking: this is importing the config module, but I guess you need an `instance` of the app config with all the processed values available in the _open_fs method.

Maybe @mvdbeek or @jmchilton know how to reference the configuration values here, I don't know a good way of doing that but I might be missing something :disappointed:

Reply:
Just don't do it. Parse the config values in the config, this is too late. `props` as serialized should contain all values ready to go.

Reply:
No worries John, we just want to do it as best as we can :)

Reply:
Thank you @jmchilton! Yes, I can rebase and fix this PR accordingly and would be happy to test it.